README.md

```diff
@@ -1,24 +1,76 @@
-# README
+# Relocate: Front-End
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Overview
 
-Things you may want to cover:
+This repo represents the front-end portion of a three-part project as part of Turing School of Software and Design's Back-End Engineering program, Module 3.  
 
-* Ruby version
+Titled Relocate, this application aims to ease the chaos of moving, providing a resource for users to discover services in their new hometown.
 
-* System dependencies
+All three repositories that make up this project can be found [here](https://github.com/Relocate08).
 
-* Configuration
+#### [Visit the Deployed Application Here](https://relocate-front-end-rails.herokuapp.com/)
 
-* Database creation
+## Table of Contents
+  - [Design](#design)
+  - [Setup](#setup)
+  - [Running the Tests](#running-the-tests)
+  - [Deployment](#deployment)
+  - [Authors](#authors)
 
-* Database initialization
+### Design
+  * The project is split into three repositories, one for Front-End, one for Back-End, and one for a Sinatra Microservice.  This repository in particular is responsible for the user-facing views along with all services and controllers necessary to query our backend's API.  
```

Comment:
We can add some key words here: split into three repositories, implementing a Service Oriented Architecture for future scalability. 
Or something like that??

Reply:
I love it, putting that in.